frontend/src/pages/restartPage/RestartPageContainer.tsx

```diff
@@ -193,6 +216,12 @@ export default ({
               "Reminded myself to tell the miniscreen to do its thing in the morning..."
             )
           )
+          .finally(() =>
+            turnOffAp && safelyRunService(
```

Comment:
At the moment, turnOffAp is only set by onConnectivityDialogSkip.

I think, instead we should always turn it off if 'shouldMoveAwayFromAp '

```suggestion
            shouldMoveAwayFromAp && safelyRunService(
```

turnOffAp state can be removed